chart/templates/post-delete-hook-configMap.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.postDelete.name}}-script
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "rancher.labels" . | indent 4 }}
+  annotations:
+    "helm.sh/hook": post-delete
+    "helm.sh/hook-weight": "1"
+    "helm.sh/hook-delete-policy": hook-succeeded
+data:
+  post-delete-hook.sh: |-
+{{ $.Files.Get "scripts/post-delete-hook.sh" | indent 4 }}
```

Comment:
Shouldn't this template rendering be conditioned by `Values.postDelete.enabled` value??

```
+{{- if .Values.postDelete.enabled }}
apiVersion: v1
kind: ConfigMap
metadata:
  name: {{ .Values.postDelete.name}}-script
  namespace: {{ .Release.Namespace }}
  labels:
{{ include "rancher.labels" . | indent 4 }}
  annotations:
    "helm.sh/hook": post-delete
    "helm.sh/hook-weight": "1"
    "helm.sh/hook-delete-policy": hook-succeeded
data:
  post-delete-hook.sh: |-
{{ $.Files.Get "scripts/post-delete-hook.sh" | indent 4 }}
+{{ end }}
```

Reply:
Yes, you are right. The condition was added.